app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -98,6 +98,12 @@ def copy
         render :json => svc.process.new_portfolio
       end
 
+      def add_icon_to_portfolio
```

Comment:
Good idea - I'll move it to a service class.

Reply:
Ended up removing the methods since an Icon can't be created without a `Iconable` type now (ie a portfolio or portfolio_item id). Therefore there is no point to have the methods.